### PR TITLE
Add support for Windows crosscompilation

### LINKED ISSUE #3: rust-embed-resource does nothing when crosscompilation for Windows is used on the Linux platform
My build script is running on a Linux machine and it basically builds executables for Windows (using crosscompilation, i.e. the target is specified explicitly), in such cases `ember-resource` does nothing and does not try to use `windres` from `MinGW` toolkit.

### PR DESCRIPTION
**Pull request by application-developer-DA**
(fixes #3)

### CODE REVIEW

File: Cargo.toml [hunk 1]
```diff
@@ -14,7 +14,8 @@ authors = ["nabijaczleweli <nabijaczleweli@gmail.com>",
            "Cat Plus Plus <piotrlegnica@piotrl.pl>",
            "Liigo <liigo@qq.com>",
            "azyobuzin <azyobuzin@users.sourceforge.jp>",
-           "Peter Atashian <retep998@gmail.com>"]
+           "Peter Atashian <retep998@gmail.com>",
+           "Daniel Abramov <dabramov@snapview.de>",]
```
Comment (nabijaczleweli) [inline Cargo.toml]: How many times have I to reiterate: don't edit my internals, especially if they're unrelated.

File: README.md [hunk 2]
```diff
@@ -30,3 +30,5 @@ In chronological order:
 [@azyobuzin](https://github.com/azyobuzin) -- providing code for finding places where RC.EXE could hide
 
 [@retep998](https://github.com/retep998) -- fixing MSVC support
+
+[@application-developer-da](https://github.com/application-developer-da) -- adding support for Windows crosscompilation (GNU toolkit)
```
Comment (nabijaczleweli) [inline README.md]: Again...

File: src/non_windows.rs [hunk 3]
```diff
@@ -1,3 +1,27 @@
-pub const SUPPORTED: bool = false;
+use std::env;
+use std::process::Command;
 
-pub fn compile_resource(_: &str, _: &str, _: &str) {}
+pub static SUPPORTED: bool = true;
```
Comment (nabijaczleweli) [inline src/non_windows.rs]: No it's bloody not; if anything. `windres` variations could go into `windows_not_msvc`.

**Comment by nabijaczleweli** [comment]
Also, use current `master` as PR base

File: Cargo.toml (hunk 1 above)
Comment (application-developer-DA) [inline Cargo.toml]: What do you mean "reiterate"? It has never been mentioned in README that you don't want contributors to add changes there. I just derived the style based on the previous changes and used them to follow the common style of repository.

File: README.md (hunk 2 above)
Comment (application-developer-DA) [inline README.md]: Ok, do you want me to remove all these mentions from both `README.md` and `Cargo.toml`?

File: src/non_windows.rs (hunk 3 above)
Comment (application-developer-DA) [inline src/non_windows.rs]: I would have done that, but the `windows_not_msvc` does not get compiled as the host platform is Linux. Any suggestions?

**Comment by nabijaczleweli** [comment]
Superseded by #13

**Comment by application-developer-DA** [comment]
NB: The PR basically the same I did in the first attempt to fix it and the final result (after refactoring) looks pretty much like the implementation from this pull request, opened in 2017. By no means that could have been done 2 years ago... (sarcasm)

**Pull request closed without merging**